scripts/EM_plot.py

```diff
@@ -0,0 +1,256 @@
+import numpy as np
+import pandas as pd
+from astropy.io import fits
+from astropy.constants import h, c, k_B, m_p
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+import mcfacts
+
+# Constants used 
+#t = 5800 #Temp of sun for now can change for what we are looking at 
+rad = 6.96e8 #Radius of sun in meters for now will change to bh later
+d = 1.5e11 #distance from earth to sun in meters 
+h = h.value #Planck constant
+c = c.value #speed of light
+k = k_B.value #Boltzmann constant
+m_p = m_p.cgs.value #proton mass
+pi = np.pi #Pi constant
+v_fs = 0.1 * c #velocity of forward shock (used in Beta)
+dens = 1e-15 #density of agn disk
+
+# Reading and convering the LSST filter data
+
+r = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_r.dat', delim_whitespace=True)
+#red filter from 5371-7059
+g = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_g.dat', delim_whitespace=True)
+# green filter from 3865-5669
+i = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_i.dat', delim_whitespace=True)
+# infrared filter from 6761-8329
+u = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_u.dat', delim_whitespace=True)
+# ultraviolet filter from 3201-4085
+y = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_y.dat', delim_whitespace=True)
+# y filter from 9085-10989
+z = pd.read_csv('src/mcfacts/inputs/data/LSST_filters/LSST_z.dat', delim_whitespace=True)
+# z filter from 8031-9385
+r_eff = r.iloc[:, 1]  # 0 for wavelength 1 for transmission
+g_eff = g.iloc[:, 1]
+i_eff = i.iloc[:, 1]  
+u_eff = u.iloc[:, 1]
+y_eff = y.iloc[:, 1]
+z_eff = z.iloc[:, 1]
+
+
+r_filter = r.iloc[:, 0]  # 0 for wavelength 1 for transmission
+g_filter = g.iloc[:, 0]
+i_filter = i.iloc[:, 0]  
+u_filter = u.iloc[:, 0]
+y_filter = y.iloc[:, 0]
+z_filter = z.iloc[:, 0]
+
+r_wave = r_filter * 1e-10 # I think its in angstroms  
+g_wave = g_filter * 1e-10
+i_wave = i_filter * 1e-10
+u_wave = u_filter * 1e-10
+y_wave = y_filter * 1e-10
+z_wave = z_filter * 1e-10
+
+# converting wavelength to frequency
+r_freq = c/r_wave #using c already below as 3e8
+g_freq = c/g_wave
+i_freq = c/i_wave
+u_freq = c/u_wave
+y_freq = c/y_wave
+z_freq = c/z_wave
+
+
+b_fs = v_fs / c
+gamma_fs = 1 / np.sqrt(1 - b_fs**2)
+
+def temp_fxn(v_fs, dens, m_p, c, k):
+    """
+    Calculate the temperature t based on velocity of forward shock and other physical parameters.
+
+    Parameters
```

Comment:
It might be helpful to someone using the function out of context to know what units these parameters should have (e.g., m/s, cm/s, K, etc.). Likewise for the other functions' parameters and outputs. Please add where relevant. A "all parameters should be in MKS/CGS units" statement in the function description would cover this nicely if that is indeed the case.